package.json

```diff
@@ -118,38 +114,39 @@
     "superagent": "^6.1.0"
   },
   "devDependencies": {
-    "@types/csurf": "^1.9.36",
-    "@types/express-session": "^1.17.0",
-    "@types/jest": "^26.0.14",
+    "@types/connect-flash": "0.0.36",
+    "@types/csurf": "^1.11.0",
+    "@types/express-session": "^1.17.3",
+    "@types/jest": "^26.0.16",
+    "@types/node": "^14.14.10",
     "@types/passport": "^1.0.4",
     "@types/superagent": "^4.1.10",
     "@types/supertest": "^2.0.10",
-    "@typescript-eslint/eslint-plugin": "^4.4.1",
-    "@typescript-eslint/parser": "^4.4.1",
+    "@typescript-eslint/eslint-plugin": "^4.9.0",
+    "@typescript-eslint/parser": "^4.9.0",
     "concurrently": "^5.3.0",
-    "cypress": "^5.4.0",
+    "cypress": "^6.0.1",
     "cypress-multi-reporters": "^1.4.0",
-    "eslint": "^7.11.0",
-    "eslint-config-airbnb-base": "^14.2.0",
-    "eslint-config-prettier": "^6.13.0",
+    "eslint": "^7.14.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-config-prettier": "^6.15.0",
     "eslint-import-resolver-typescript": "^2.3.0",
     "eslint-plugin-cypress": "^2.11.2",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-prettier": "^3.1.4",
     "husky": "^4.3.0",
-    "jest": "^26.5.3",
+    "jest": "^26.6.3",
     "jest-html-reporter": "^3.3.0",
-    "jest-junit": "^11.1.0",
+    "jest-junit": "^12.0.0",
     "jsonwebtoken": "^8.5.1",
-    "lint-staged": "^10.4.1",
-    "mocha": "^8.1.3",
+    "lint-staged": "^10.5.2",
     "mocha-junit-reporter": "^2.0.0",
```

Comment:
Is there any point using this if we're don't have mocha any more or is it used for something else?

Reply:
Based on this cypress config:

```
{
  "reporterEnabled": "spec, mocha-junit-reporter",
  "reporterOptions": {
    "mochaFile": "test_results/cypress/results-[hash].xml"
  }
}
```

this is actually used by cypress to generate junit test result files in this location:

`test_results/cypress/results-some-hash.xml`

These are then uploaded to cypress through this config: 
```
      - store_test_results:
          path: test_results
```

The results can then be seen in the circle ci ui:
<kbd>
<img width="952" alt="Screenshot 2020-12-07 at 08 29 55" src="https://user-images.githubusercontent.com/1517745/101327321-707d1f80-3866-11eb-8d37-e529022ffd8e.png">
</kbd>

(Fixed the file path so this actually works!)